pkg/webhooks/admission/jobs/validate/admit_job.go

```diff
@@ -274,3 +278,17 @@ func validateTaskTemplate(task v1alpha1.TaskSpec, job *v1alpha1.Job, index int)
 
 	return ""
 }
+
+func validateK8sPodNameLength(podName string) string {
+	if errMsgs := validation.IsDNS1123Label(podName); len(errMsgs) > 0 {
+		return fmt.Sprintf("create pod with name %s validate failed %v;", podName, errMsgs)
+	}
+	return ""
+}
+
+func validateK8sSvcNameLength(job *v1alpha1.Job) string {
```

Comment:
It suddenly occurred to me that these two validation may be found in k8s.io

Reply:
validation.IsDNS1123Label  method ？

Reply:
I think we just need to  validate podname and svcname by validation.IsDNS1123Label method

Reply:
Please go fmt midified files and make CI happy

Reply:
> I think we just need to validate podname and svcname by validation.IsDNS1123Label method

`IsQualifiedName` method may be a good choice ?

Reply:
Ok, i have updated

Reply:
Now, do you have any other questions? @Thor-wl